lib/navigation/consts.hpp

```diff
@@ -0,0 +1,47 @@
+#include <cstdint>
+
+namespace hyped::navigation{
+
+  using nav_t = float;
+  static constexpr nav_t kTrackLength       = 100.0; //m
+  static constexpr nav_t kBrakingDistance   = 20.0;  //m
+  static constexpr uint8_t kNumImus         = 4;
+  static constexpr uint8_t kNumEncoders     = 4;
+  static constexpr uint8_t kNumKeyence      = 2;
+  static constexpr nav_t kPi                = 3.14159265359;
+  static constexpr nav_t kWheelCicumference = kPi * 0.1; //m //TODO: check!
+  static constexpr nav_t kStripeDistance    = 10.0; //m
+  
+  //current trajectory struct
+  struct Trajectory {
```

Comment:
This probably wants to live somewhere else? E.g. state machine and telemetry will want to use this type and they don't want to link against navigation.